test/test_dist_autograd.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import unittest
+import torch.distributed.autograd as dist_autograd
+
+class TestDistAutograd(unittest.TestCase):
+
+    def test_autograd_context(self):
+        worker_id = 16
+        dist_autograd.init(worker_id)
```

Comment:
if `dist_autograd` always work with `RPC` (I assume so, as it needs send/recv), can we merge this with `init_rpc`, so that users don't have to call two init API? For example, it could be `init_rpc` by default also initialize dist autograd, but there could an arg to disable it if the application won't run any backward on top of RPC.